tools/python/maps_generator/utils/file.py

```diff
@@ -189,7 +189,7 @@ def is_verified(name: AnyStr) -> bool:
 def copy_overwrite(from_path: AnyStr, to_path: AnyStr):
     if os.path.exists(to_path):
         shutil.rmtree(to_path)
-    shutil.copytree(from_path, to_path)
```

Comment:
if `from_path` points to a file (not a directory), then `shutil.copytree` fails with exception. Replaced with `shutil.copy`